<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCP_Server_A</class>
 <widget class="QMainWindow" name="TCP_Server_A">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCP_Server_A</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="openPushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Button</string>
    </property>
   </widget>
   <widget class="QLabel" name="sendStatusLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="sendProgressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendPushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Button</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectHost">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect To Host</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectHost_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy File</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
